src/models/Subscriber.ts

```diff
@@ -0,0 +1,25 @@
+import mongoose, { Document, Schema } from 'mongoose';
+
+interface ISubscriber extends Document {
+    email: string;
+    dateSubscribed: Date;
+}
+
+const subscriberSchema: Schema = new Schema({
+    email: {
+        type: String,
+        required: true,
+        unique: true,
+        trim: true,
+        lowercase: true,
+    },
+    dateSubscribed: {
+        type: Date,
+        required: true,
+        default: Date.now
+    }
```

Comment:
remove this field and add a timestamps: true to the config of the model. 

https://mongoosejs.com/docs/timestamps.html